src/utils/utils.ts

```diff
@@ -18,7 +18,7 @@ export class Utils {
 	};
 
 	static debounce = (func: () => void, delay = 500) => {
-		let timeout: number;
+		let timeout: ReturnType<typeof setTimeout>;
```

Comment:
I think this worked fine as it was.

Reply:
this was to resolve type after getting `@types/node`